src/Microsoft.Identity.Client/ITokenCache.cs

```diff
@@ -27,16 +27,15 @@
 
 using Microsoft.Identity.Client.Cache;
 
-namespace Microsoft.Identity.Client.CacheV2
+namespace Microsoft.Identity.Client
 {
     /// <summary>
     /// This is the interface that implements the public access to cache operations.
     /// With CacheV2, this should only be necessary if the caller is persisting
     /// the cache in their own store, since this will provide the serialize/deserialize
     /// and before/after notifications used in that scenario.
-    /// TODO: This interface should eventually be PUBLIC.
     /// </summary>
-    internal interface ITokenCache
+    public interface ITokenCache
```

Comment:
@MarkZuber : one issue to be able to build the updated samples, is that for the moment TokenCache does not implement ITokenCache.

Reply:
Can I do that in an immediate follow up PR so as to not bloat this one further?

---
In reply to: [246272750](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/761#discussion_r246272750) [](ancestors = 246272750)

Reply:
will do this in the next pr

---
In reply to: [246491786](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/761#discussion_r246491786) [](ancestors = 246491786,246272750)